Pinta.Tools/CoreToolsExtension.cs

```diff
@@ -104,6 +104,8 @@ public void Uninitialize ()
 			PintaCore.Tools.RemoveInstanceOfTool (typeof (EllipseTool));
 			PintaCore.Tools.RemoveInstanceOfTool (typeof (FreeformShapeTool));
 		}
-#endregion
+		#endregion
 	}
+
+	public delegate void MouseHandler (double x, double y, Gdk.ModifierType state);
```

Comment:
Is this still used anywhere?

Reply:
Yes, it's used by `ToolControl.cs`.  It would be nice if maybe it could be refactored and removed in the future.

Reply:
Ah right, I didn't see references to it in the diff.
From the looks of it, the `MouseMoved` etc events on `BaseTool` aren't used anywhere, so it could probably be moved directly into `ToolControl`